lib/utils/validate.ts

```diff
@@ -26,7 +26,7 @@ export function encodePathParams<TParams extends {}>(params: TParams) {
 }
 
 export function getPathParam(value: string) {
-    return /^((?!(\.\.|\?|#|\\|\/)).)*$/i.test(value) ? value : '';
+    return /^((?!(\.\.|\?|#|\\|\/)).)*$/i.test(value) ? value : 'INVALID_PARAM_VALUE';
```

Comment:
What if we use 'GATEWAY_INVALID_PARAM_VALUE' to make it more specific, to easy recognize that it comes from gateway package?

Reply:
Good idea